holidays/countries/mali.py

```diff
@@ -0,0 +1,146 @@
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import JAN, MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Mali(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolidays):
+    """Mali holidays.
+
+    References:
+        * [Ordinance no. 54 of 1960](https://web.archive.org/web/20250619193041/https://sgg-mali.ml/JO/1960/mali-jo-1960-71.pdf)
+        * [Law No. 28 of 1964](https://web.archive.org/web/20250619190638/https://sgg-mali.ml/JO/1964/mali-jo-1964-183.pdf)
+        * [Ordinance no. 16 of 1992](https://web.archive.org/web/20250611221453/https://sgg-mali.ml/JO/1992/mali-jo-1992-08.pdf)
+        * [Law No. 40 of 2005](https://web.archive.org/web/20250603120027/https://sgg-mali.ml/JO/2005/mali-jo-2005-25.pdf)
+    """
+
+    country = "ML"
+    default_language = "fr"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    start_year = 1961
+    supported_languages = ("en_US", "fr")
+
+    def __init__(self, *args, islamic_show_estimated: bool = True, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=MaliIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Jour de l'An"))
+
+        if self._year >= 1965:
+            # Armed Forces Day.
+            self._add_holiday_jan_20(tr("Journée de l'Armée"))
+
+        if self._year >= 1992:
+            # Martyrs' Day.
+            self._add_holiday_mar_26(tr("Journée du 26 mars"))
+
+        if self._year >= 2006:
+            # Easter Monday.
+            self._add_easter_monday(tr("Lundi de Pâques"))
+
+        # Labor Day.
+        self._add_labor_day(tr("Fête du Travail"))
+
+        if self._year >= 1965:
+            # Africa Day.
+            self._add_africa_day(tr("Journée de l'Afrique"))
+
+        # National Day of the Republic of Mali.
+        self._add_holiday_sep_22(tr("Fête Nationale de la République du Mali"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Fête de Noël"))
+
+        # Prophet's Birthday.
+        self._add_mawlid_day(tr("Journée du Mawloud"))
+
+        if self._year >= 2006:
+            # Prophet's Baptism.
+            self._add_prophet_baptism_day(tr("Journée du Baptême"))
```

Comment:
Maybe it would be better to use something like "Journée du Maouloud (Naissance du Prophète)" and "Journée du Maouloud (Baptême du Prophète)"? This is closer to the official version and is quite clear at the same time.
To be more precise, name “Journée du Mawloud” can be used for Prophet's Birthday before 2006.